doc/docs/getting-started/docker.md

```diff
@@ -8,7 +8,7 @@ Leek is a full stack application built using different technologies and publishe
 [public repository](https://hub.docker.com/repository/docker/kodhive/leek) on DockerHub
 
 Leek is a multi-services application. during docker container startup, supervisord will start as the process with pid=1
-and will start the enabled service and skip the disabled ones relying on `LEEK_ENABLE_AGENT`, `LEEK_ENABLE_AGENT`, 
```

Comment:
Thanks @haim0n for your contribution. First contributor 🎉 

The dup string `LEEK_ENABLE_AGENT` should be replaced by `LEEK_ENABLE_API`

Reply:
Done :)

kudos for an awesome project !